app.js

```diff
@@ -243,7 +243,7 @@ function App (el) {
         self.emit('leaveChannel', self.data.activeChannel.name)
         break
       default:
-        console.log('Unrecognized /command: ' + command)
+        console.log('Unrecognized command: ' + commandStr)
```

Comment:
should still be `command`, no? `commandStr` is unprocessed by https://github.com/tunnckoCore/friends/blob/master/app.js#L232-L234